modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationAfterAccountDisablingTestCase.java

```diff
@@ -0,0 +1,445 @@
+        tenantIDPMgtClient = new IdentityProviderMgtServiceClient(tenantCookie, backendURL);
+    }
+
+    @Test(description = "Create access tokens")
+    public void testCreateAccessTokens() throws Exception {
+
+        Set<String> appKeys = applications.keySet();
+        for (String appName : appKeys) {
+            OAuthConsumerAppDTO appDTO = applications.get(appName);
+            ClientID consumerKey = new ClientID(appDTO.getOauthConsumerKey());
+            Secret consumerSecret = new Secret(appDTO.getOauthConsumerSecret());
+            AccessToken accessToken = requestAccessToken(consumerKey, consumerSecret);
+            accessTokens.put(appName, accessToken);
+            AccessToken privilegedAccessToken = requestPrivilegedAccessToken(consumerKey, consumerSecret);
+            privilegedAccessTokens.put(appName, privilegedAccessToken);
+
+            // Introspect the returned access token to verify the validity.
+            TokenIntrospectionResponse activeTokenIntrospectionResponse = introspectAccessToken(accessToken,
+                    privilegedAccessToken);
+            Assert.assertTrue(activeTokenIntrospectionResponse.indicatesSuccess(),
+                    "Failed to receive a success response.");
+            Assert.assertTrue(activeTokenIntrospectionResponse.toSuccessResponse().isActive(),
+                    "Introspection response of an active access token is unsuccessful.");
+        }
+    }
+
+    @Test(description = "Enabling the user account disabling feature for resident IDP",
+            dependsOnMethods = "testCreateAccessTokens")
+    private void enableUserAccountDisablingFeature() throws Exception {
+
+        IdentityProviderProperty[] idpProperties = residentIDP.getIdpProperties();
+        for (IdentityProviderProperty providerProperty : idpProperties) {
+            if (ENABLE_ACCOUNT_DISABLING_PROPERTY.equalsIgnoreCase(providerProperty.getName())) {
+                providerProperty.setValue("true");
+            }
+        }
+        updateResidentIDP(residentIDP, true);
+    }
+
+    @Test(description = "Disabling the test user account", dependsOnMethods = "enableUserAccountDisablingFeature")
+    private void disableUserAccount() throws Exception {
+
+        setUserClaim(ACCOUNT_DISABLED_CLAIM_URI, "true");
+        ClaimValue[] claimValues = usmClient.getUserClaimValuesForClaims(TEST_USER_USERNAME, new String[]
+                {ACCOUNT_DISABLED_CLAIM_URI}, "default");
+        String accountDisabledClaimValue = null;
+        if (ArrayUtils.isNotEmpty(claimValues)) {
+            accountDisabledClaimValue = claimValues[0].getValue();
+        }
+        Assert.assertTrue(Boolean.parseBoolean(accountDisabledClaimValue), "User account didn't disabled");
+    }
+
+    @Test(description = "Check whether access token is revoked after disabling the account",
+            dependsOnMethods = "disableUserAccount")
+    private void introspectAccessTokenOfDisabledAccount() throws Exception {
```

Comment:
End the method name with "test"

Reply:
Fixed in https://github.com/wso2/product-is/pull/14727/commits/4882f9bbdb620ceaaf5e04a15be585a6c1676420